content/go/concepts/math-functions/terms/mod/mod.md

```diff
@@ -0,0 +1,180 @@
+---
+Title: 'Mod()'
+Description: 'Returns the floating-point remainder of dividing "x" by "y".'
+Subjects:
+  - 'Code Foundations'
+  - 'Computer Science'
+Tags:
+  - 'Arithmetic'
+  - 'Functions'
+  - 'Numbers'
+CatalogContent:
+  - 'learn-go'
+  - 'paths/computer-science'
+---
+
+The **`Mod()`** function returns the floating-point remainder of dividing **x** by **y**. The result has the same sign as **x** and a magnitude less than the magnitude of **y**.
+
+Mod function operates exclusively with floating-point numbers.
+If the numbers involved in the operation are of integer type, the **%** modulus operator should be used instead. The % operator works with integers, while math.Mod operates with floating-point numbers.
+
+## Syntax
+
+```pseudo
+math.Mod(x,y)
+```
+
+The `Mod()` function takes two arguments of type 'float64'.
+
+<!---blank--->
+
+
+### Some special cases:
+1. If the argument has a **+Inf** value, the return value will be **0**:
```

Comment:
```suggestion
1. If the argument has a `+Inf` value, the return value will be `0`:
```

Reply:
Each of these is unnecessarily long. For example, (for number 1) keep either line 33 or 35, not both. The general aim of docs content is to provide clear and very concise explanations.


Edit the other numbers accordingly.